wiki/People/Beatmap_Nominators/Rules/en.md

```diff
@@ -47,7 +47,7 @@ Violations of those rules may be reported to a member of the [Nomination Assessm
 + **Ensure that kudosu is awarded to all posts that deserve it.** Additionally consult the mapper before giving out kudosu to make sure there are not any reasons why the post does not need kudosu (e.g. the modder has received one already for a different post which you overlooked). If the modpost is too old to receive kudosu, consult the modder to make an additional post linking the original he receives kudosu for.
 + **You must not place any icons besides the star, bubble, bubble pop and heart.** Whilst you are able to set other icons, they are either deprecated and discouraged from use (flame, circle, drum, apple, piano) or reserved for use by the Nomination Assessment Team (nuke). If you accidentally set an unwanted icon, contact a NAT immediately or overwrite the icon in case you wanted to post a bubble or heart.
 + **Do not nominate a beatmap without properly checking all difficulties of the beatmap all by yourself.** If you are unfamiliar with the game mode some of the difficulties are mapped in, ask fellow Beatmap Nominators for help. Improper checking includes just running a modding tool before nominating or not even downloading the beatmap.
-+ **You can veto a nominated/qualified beatmap set by posting the issues you think need to be addressed before reaching Ranked status in the form of problems in the respective modding discussion.** This will reset all nominations the beatmap set has received and thereby stop it from proceeding to Ranked. Other nominators will be unable to nominate it again until the veto is lifted. The veto post must include a clear problem and reasoning for stopping the ranking process of the beatmap set. You must participate in the resulting discussion and attempt to reach a mutual agreement.
-+ **Do not veto for unrankables.** Unrankables in themsleves mean that no other nominator should nominate and that the mapper has no other choice but to fix it, in the same way an upheld veto works. Because of this, disqualifying for unrankables often requires a second person to confirm the issue.
++ **You can veto a nominated/qualified beatmap set by posting the issues you think need to be addressed before reaching Ranked status in the form of problems in the respective modding discussion.** This will reset all nominations the beatmap set has received and thereby stop it from proceeding to Ranked. Other nominators will be unable to nominate it again until the veto is lifted. The veto post must clearly state that it is a veto and include a clear problem and reasoning for stopping the ranking process of the beatmap set. You must participate in the resulting discussion and attempt to reach a mutual agreement.
```

Comment:
```suggestion
+ **You can veto a nominated/qualified beatmapset by posting the issues you think need to be addressed before reaching Ranked status in the form of problems in the respective modding discussion.** This will reset all nominations the beatmapset has received and thereby stop it from proceeding to Ranked. Other nominators will be unable to nominate it again until the veto is lifted. The veto post must clearly state that it is a veto and include a clear problem and reasoning for stopping the ranking process of the beatmapset. You must participate in the resulting discussion and attempt to reach a mutual agreement.
```
beatmap set => beatmapset as decided in RC a while ago